FollowUpAppointment/README.md

```diff
@@ -14,15 +15,18 @@ Let your mobile workers create and schedule follow-up appointments for customers
 		- Field Service Resource License
 		- Field Service Resource Permissions
 		- Field Service Follow-Up Appointment Permissions
-4. Grant access to Apex classes.
+3. Grant access to Apex classes.
 	1. From Setup, in the Quick Find box, enter `Custom Code`, and then select **Apex Classes**.
 	2. For FollowUpAppointmentController, click **Security**.
 	3. Add the relevant profiles to the Enabled Profiles list.
+4. To install the package, click 
+   https://login.salesforce.com/packaging/installPackage.apexp?p0=04t1Q0000012BwaQAE.
 5. From the App Launcher, find and select **Follow-Up Appointment Settings**.
 6. Configure the LWC.
 	1. In the Settings tab, determine which appointment slots the mobile worker sees and determine who the work can be assigned to and what kind of work they can create.
-If you choose to let mobile workers create the service appointment, only a service appointment is created. If you choose to let mobile workers create work orders or work order line items, a service appointment is created automatically. A service appointment is created regardless of the Auto-Create Service Appointment setting for the work type.
-*Tip:* To control whether the service appointment can be assigned only to the mobile worker in the app, make sure that your scheduling policy includes the **Field Service - Required Resources work rule**.
+	If you choose to let mobile workers create the service appointment, only a service appointment is created. If you choose to let mobile workers create work orders or work order line items, a service appointment is created automatically. A service appointment is created regardless of the Auto-Create Service Appointment setting for the work type.
+
+	*Tip:* To control whether the service appointment can be assigned only to the mobile worker in the app, make sure that your scheduling policy includes the **Field Service - Required Resources work rule**.
 	2. In the Mobile Workflow tab, configure up to four steps. 
```

Comment:
This item is getting inlined with the text block above. I think you have to have a blank line between the "block-quote" and this next entry, to avoid them being concatenated.
![image](https://github.com/forcedotcom/LWC-Mobile-Samples/assets/755462/48adc522-76c3-4f19-83bc-650d4c25d5fc)